gen/dcos-config.yaml

```diff
@@ -698,6 +698,21 @@ package:
           "LocalFiles": [
               {
                   "Location": "/opt/mesosphere/active.buildinfo.full.json"
+              },
+              {
+                  "Location": "/opt/mesosphere/active/dcos-metadata/etc/dcos-version.json"
```

Comment:
Is there a reason we shouldn't use `/opt/mesosphere/etc/dcos-version.json` instead? If it's all the same, we might as well use the path that won't break if we change the name of the package that provides this file.